src/app/pages/ArticlePage/Byline/index.stories.tsx

```diff
@@ -2,33 +2,42 @@ import React, { PropsWithChildren } from 'react';
 import { withKnobs } from '@storybook/addon-knobs';
 import { ServiceContextProvider } from '../../../contexts/ServiceContext';
 import Timestamp from '../../../legacy/containers/ArticleTimestamp';
-import { bylineWithNameAndRole } from './fixture';
+import { bylineWithNameAndRole, bylineWithLink } from './fixture';
 import Byline from '.';
 import { withServicesKnob } from '../../../legacy/psammead/psammead-storybook-helpers/src';
+import ThemeProvider from '../../../components/ThemeProvider';
 
 interface Props {
-  service: string;
+  service: any;
```

Comment:
You can import the `Services` type from `src/app/models/types/global.ts` and use it here.

Reply:
Done